rest/test/plugins/routes/receiptsRoutes_spec.js

```diff
@@ -44,12 +47,11 @@ describe('receipts routes', () => {
 			}
 		};
 
-		const highestHeight = 50;
 		receiptsRoutes.register(server, {
-			chainInfo: () => Promise.resolve({ height: highestHeight }),
-			transactionStatementsAtHeight: statementsFakes[0],
-			addressResolutionStatementsAtHeight: statementsFakes[1],
-			mosaicResolutionStatementsAtHeight: statementsFakes[2]
+			catapultDb: {
+				chainInfo: () => Promise.resolve({ height: highestHeight }),
```

Comment:
Unneeded trailing comma.

```suggestion
				chainInfo: () => Promise.resolve({ height: highestHeight })
```